### PR TITLE
fix description for module

### PR DESCRIPTION
**Pull request by trifonovmixail**

### CODE REVIEW

File: pytest_allure_dsl.py [hunk 1]
```diff
@@ -45,13 +45,13 @@ def allure_dsl(request):
 @pytest.fixture('function', autouse=True)
 def __allure_dls_pre_post_actions__(request, allure_dsl):
     if request.config.option.allure_dsl:
-        MASTER_HELPER.description(allure_dsl.description)
+        allure_dsl.setup()
```
Comment (griddic) [inline pytest_allure_dsl.py]: May be context manager would be more suitable instead of .setup() and .teardown() ?

File: pytest_allure_dsl.py [hunk 2]
```diff
@@ -45,13 +45,13 @@ def allure_dsl(request):
...
 
         try:
             yield
         finally:
             if request.config.option.allure_dsl:
```
Comment (griddic) [inline pytest_allure_dsl.py]: request.config.option.allure_dsl may be changed during execution?

File: pytest_allure_dsl.py (hunk 1 above)
Comment (trifonovmixail) [reply pytest_allure_dsl.py]: Done

File: pytest_allure_dsl.py (hunk 2 above)
Comment (trifonovmixail) [reply pytest_allure_dsl.py]: Done

File: setup.py [hunk 3]
```diff
@@ -14,7 +14,7 @@
     py_modules=['pytest_allure_dsl'],
     install_requires=[
         'pytest',
-        'pytest-allure-adaptor==1.7.10',
+        'git+https://github.com/oztqa/allure-pytest.git@1.7.11',
```
Comment (griddic) [inline setup.py]: error in pytest-allure-dsl setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; Invalid requirement, parse error at "'+https:/'"

**Review by griddic — APPROVED** [review]
verified on pytest==3.6.0

**Pull request merged**